components/dialog/dialog-mixin.js

```diff
@@ -282,9 +282,11 @@ export const DialogMixin = superclass => class extends RtlMixin(superclass) {
 	_handleKeyDown(e) {
 		if (!this.opened) return;
 		if (e.keyCode === 27) {
-			// escape (note: prevent native dialog close so we can: animate it; use setDismissible)
+			// escape (note: prevent native dialog close so we can: animate it;)
 			e.stopPropagation();
 			e.preventDefault();
+			if (!this.opened) return;
+			this._close(abortAction);
```

Comment:
Now that the dismissible code is listening for `keydown` as well, it no longer gets called here -- so we need to run the code to close the dialog ourselves.

Reply:
Do we just need to remove the `e.stopPropagation();` so that the dismissible event handler can run?  Otherwise we're setting up a dismissible callback and not ever using it.

Reply:
🤦  Yep that is indeed all we need! Much cleaner now.